src/MessagePack.UnityClient/Assets/Scripts/MessagePack/MessagePackSecurity.cs

```diff
@@ -334,8 +334,8 @@ public override unsafe int GetHashCode(float value)
                     value = float.NaN;
                 }
 
-                long l = *(long*)&value;
-                return HashCode.Combine((int)(l >> 32), unchecked((int)l));
+                int l = *(int*)&value;
+                return l;
```

Comment:
This looks like an important bug fix that we should include in `master` as well. Would you mind filing that PR separately and target `master`?
Was there anything about this PR that requires this fix to be included? If not, maybe drop it from this PR and we'll merge it into `develop` from `master`.

Reply:
I had to include this bug fix in this PR in order to pass the CI.
I will make another separate PR on master branch.